### ISSUE TITLE
Version 2.8.0 (on F-Droid) causes system crash and boot loops

### ISSUE THREAD

**Issue by HennesTheSecond** [PrivacyApps/document-viewer#273]
Unfortunately the f-droid build version 2.8.0 instantly crashes my system (Nexus 10, LineageOS 13 official nightly) after installation and forces my device into bootloops. Multiple other apps and devices are affected and more and more users will run into the same issue when installing the offered update through F-Droid.

Please read this F-Doid issue for further details:
https://gitlab.com/fdroid/fdroiddata/issues/979

If you ran into the same problems use the "safe mode" to boot into android and uninstall Document Viewer 2.8.0.
On Nexus phones you have to hold the "volume down" button during start to enter safe mode. (https://support.google.com/nexus/answer/2852139?hl=en)

@dschuermann: Is there a way to delete 2.8.0 from f-droid till they have fixed their build issue?

Beside of that the F-Droid page of Document Viewer looks a bit strange to me:
https://f-droid.org/packages/org.sufficientlysecure.viewer/
Version 2.8.0 is listed six times there ...

**Comment by ericwa** [comment]
> Version 2.8.0 is listed six times there ...

This is because we use a separate APK per ABI. I think this is just for space saving reasons; mupdf compiles fonts into the library so the APK would be huge otherwise.

It sounds like a new release with build tools 27.0.1 will work around the issue, according to: https://gitlab.com/fdroid/fdroiddata/merge_requests/2618#note_49069444 and the linked Google issue.

**Comment by ericwa** [comment]
I made a 2.8.1 release with the build tools update. It's merged into fdroiddata, just need to wait for the next rebuild and then see if the problem is fixed.

**Comment by ericwa** [comment]
Our side of the workaround is done so I'll close this.
Getting document-viewer 2.8.1 to appear in fdroid is currently blocked by: https://gitlab.com/fdroid/fdroidserver/issues/381
